apps/governance/src/routes/proposals/components/proposal/proposal.tsx

```diff
@@ -32,6 +32,7 @@ export interface ProposalProps {
   proposal: ProposalFieldsFragment | ProposalQuery['proposal'];
   networkParams: Partial<NetworkParamsResult>;
   newMarketData?: MarketInfoWithData | null;
+  parentMarketData?: MarketInfoWithData | null;
   assetData?: AssetQuery | null;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   restData: any;
```

Comment:
**nitpick (non-blocking)**: We probably brought this up when it was introduced, and it wasn't introduced here so it's not a change request. But it *is* possible to type the REST responses. Block explorer has types generated. The reason it's not a blocker is that it still doesn't guarantee that the underlying REST responses matches the type, so it's more the illusion of proper typing. More so than normal.

Reply:
Oh, thanks, I'll deal with this separately but I'll check out the Explorer code.